br/pkg/lightning/backend/local/localhelper.go

```diff
@@ -399,6 +399,14 @@ func (local *Backend) SplitAndScatterRegionByRanges(
 	return nil
 }
 
+func (local *Backend) ScatterRegion(ctx context.Context, regionInfo *split.RegionInfo) error {
+	backoffer := split.NewWaitRegionOnlineBackoffer().(*split.WaitRegionOnlineBackoffer)
+	_ = utils.WithRetry(ctx, func() error {
+		return (local.splitCli.ScatterRegion(ctx, regionInfo))
```

Comment:
Oh I understand now. We should retry for multiple times when `ScatterRegion` fails. Will take a look later

Reply:
did you get chance to look into it

Reply:
plz add a warn log on err, it helps diagnose, and the err need to be wrapped for now, see

https://github.com/pingcap/tidb/blob/64a2e1c20a0666f8f8c4ae46330dedcc406a5d47/br/pkg/lightning/backend/local/localhelper.go#L421-L425

Reply:
fixed.